src/crypto/tls_mbedtls_alt.c

```diff
@@ -64,8 +64,6 @@
 #include <mbedtls/debug.h>
 #ifdef __ZEPHYR__
 #define PRINTF printk
-#else
-#include "fsl_debug_console.h"
```

Comment:
isn't this for non-zephyr? commit log says it's unused in Zephyr?

Reply:
"fsl_debug_console.h" is a NXP header file, should be removed here.

Reply:
I got that, but why was it used for non-zephyr branch, was it intentional or mistake? Just trying to understand the original change.

Reply:
yes, it was used for non-zephyr branch and it's intentional to support printf.
but adding here is mistake.